src/main/java/com/globalitgeeks/examninja/usermanagement/controller/UserController.java

```diff
@@ -32,7 +32,15 @@ public ResponseEntity<?> register(@RequestBody UserRequest userRequest) {
     }
 
     // Login Endpoint
-
+     @PostMapping("/login")
+    public ResponseEntity<?> login(@RequestBody LoginRequest loginRequest) {
+        try {
+            User user = userService.login(loginRequest.getEmail(), loginRequest.getPassword());
+            return ResponseEntity.ok(user);
+        } catch (Exception e) {
+            return ResponseEntity.status(401).body(e.getMessage());
```

Comment:
Use ControllerAdvice for handling exception.

Reply:
using @RestControllerAdvice for GlobalExceptionHandling.class  i exception package.

Reply:
Addressed in recent commit